lib/ansible/modules/cloud/lxd/lxd_container.py

```diff
@@ -381,6 +388,8 @@ def _unfreeze_container(self):
         self.actions.append('unfreez')
 
     def _container_ipv4_addresses(self, ignore_devices=['lo']):
+        if self.wait_for_ipv4_addresses_ignore_devices:
+            ignore_devices = ['lo'] + self.wait_for_ipv4_addresses_ignore_devices
```

Comment:
Thanks.  I see your point.  Definitely missed that string thing though.  Changes pushed.  ;)